### PR TITLE
Contributor section updates

### PR DESCRIPTION
**Pull request by sheridancook**

### CODE REVIEW

**Comment by sheridancook** [comment]
@ElliotSilver - you ok being secondary reviewer on these to merge them in (format discussed and approved on 8/30 governance call)

File: input/pagecontent/index.md [hunk 1]
```diff
@@ -54,58 +54,14 @@ Guidance, Capability Statements, and other have not yet been reviewed and added.
...
-| **Contributor** | Ken Sinn |
-| **Contributor** | Janice Spence |
-| **Contributor** | Erik Uittien |
-| **Contributor** | Tony Waldschmidt |
-| **Contributor** | Thomas Zhou |
 
 
+| Role  | Role Description| Name |
+| --- | --- | -- |
+| **Primary Authors** | Anyone historically involved in the  authoring/development of the spec (including stream hosts) |Russ Buchanan, &nbsp;&nbsp; Sheridan Cook, &nbsp;&nbsp; Shamil Nizamov, Scott Prior, &nbsp;&nbsp; Michael Savage, &nbsp;&nbsp; Elliot Silver, &nbsp;&nbsp; Igor Sirkovich |
```
Comment (ElliotSilver) [inline input/pagecontent/index.md:61]: Is "Anyone" the best word? "People"?  (If switching make sure verb number agrees, i.e. "People who have pushed...")

File: input/pagecontent/index.md [hunk 2]
```diff
@@ -54,58 +54,14 @@ Guidance, Capability Statements, and other have not yet been reviewed and added.
...
+| **Contributors** | Participants in the Entities, Medications, and Clinical Profiling Sub-Streams, Vocal contributors in governance and/or profiling calls, issue submitters | Ian Bekker, &nbsp;&nbsp; Anne Belford, &nbsp;&nbsp; Alex Bennett, &nbsp;&nbsp; Michael Boivin, &nbsp;&nbsp; Fang Cao, &nbsp;&nbsp; Lorraine Constable, &nbsp;&nbsp; Ramandeep Dhanoa, &nbsp;&nbsp; Jean Duteau, &nbsp;&nbsp; Finnie Flores, &nbsp;&nbsp; Joel Francis, &nbsp;&nbsp; Alex Goel, &nbsp;&nbsp; Joanie Harper, &nbsp;&nbsp; Maria Hu, &nbsp;&nbsp; Anibal Jodorcovsky, &nbsp;&nbsp; Mukesh Kashyap, &nbsp;&nbsp; Paul Knapp, &nbsp;&nbsp; Marc L'Arrivee, &nbsp;&nbsp; Francis Lau, &nbsp;&nbsp; Adesh Maharaj, &nbsp;&nbsp; Dean Matthews, &nbsp;&nbsp; Randy Nonay, &nbsp;&nbsp; Ron Parker, &nbsp;&nbsp; Anil Patel, &nbsp;&nbsp; Melva Peters, &nbsp;&nbsp; Rita Pyle, &nbsp;&nbsp; Derek Ritz, &nbsp;&nbsp; Philip Sales, &nbsp;&nbsp; Khushwinder Sekhon, &nbsp;&nbsp; Harsh Sharma, &nbsp;&nbsp; Sisira De Silva, &nbsp;&nbsp; Dan Simic, &nbsp;&nbsp; Ken Sinn, &nbsp;&nbsp; Janice Spence, &nbsp;&nbsp; Erik Uittien, &nbsp;&nbsp; Tony Waldschmidt, &nbsp;&nbsp; Thomas Zhou |
```
Comment (ElliotSilver) [inline input/pagecontent/index.md:63]: "Sub-streams, Vocal contributors... calls, issue submitters": replace those commas with semicolons.

Also, consider replacing commas in name list with semicolons to allow "Paul Knapp, FHL7; Marc L'Arrivee, MD; Francis Lau, University of Victoria;..."

**Review by ElliotSilver — APPROVED** [review]
Approved with some suggestions.

**Pull request merged**